CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ IMPROVEMENTS:
   * Allow customization of `terminationGracePeriodSeconds` on the ingress gateways. [[GH-947](https://github.com/hashicorp/consul-k8s/pull/947)]
   * Support `ui.dashboardURLTemplates.service` value for setting [dashboard URL templates](https://www.consul.io/docs/agent/options#ui_config_dashboard_url_templates_service). [[GH-937](https://github.com/hashicorp/consul-k8s/pull/937)]
   * Allow using dash-separated names for config entries when using `kubectl`. [[GH-965](https://github.com/hashicorp/consul-k8s/pull/965)]
+* Control Plane
+  * Support the value `$POD_NAME` for the annotation `consul.hashicorp.com/service-meta-pod_name` that will now be interpolated and set to the pod name in the service's metadata. [[GH-982](https://github.com/hashicorp/consul-k8s/pull/982)]
```

Comment:
@thisisnotashwin it's supported for any meta annotation right? Not just `service-meta-pod_name` but also `service-meta-foo` for example.

Reply:
not really. I asked @david-yu and the ask from the customer is to only support `$POD_NAME`.

Reply:
or would it make sense to add another value for `$POD_NAMESPACE`?

Reply:
It's also important to note that this isn't actually an env var. We're doing this in the endpoints controller that doesn't have access to the actual pod's environment variables. So we can only support data we actually have when looking at the pod yaml.

Reply:
Got it, would it be possible to add all that data that is available on the pod (e.g, `namespace`)?